content/en/blog/_posts/2022-12-6-provision-volumes-from-cross-namespace-snapshots-alpha.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: blog
+title: "Kubernetes 1.26: alpha support for provision volumes from cross-namespace snapshots"
+date: 2022-12-6
+slug: provision-volumes-from-cross-namespace-snapshots-redesigned
+---
+
+**Authors:**
+Takafumi Takahashi (Hitachi Vantara)
+
+Kubernetes v1.26, released earlier this month, introduced an enable usage of provision of persisten volume claim from volume snapshot in other namespaces.  
+Before Kubernetes v1.25, by using volume snapshots feature, users can provision volumes from snapshots. However, it only works for the `VolumeSnapshot` in the samme namespace, therefore users can't provision a persisten volume claim in one namespace from a `VolumeSnapshot` in other namespace. On the other hand, there are use cases that require to share the `VolumeSnapshot` across namespaces.  
+ To solve this problem,Kubernetes v1.26 includes a new API field called `dataSourceRef2`.
```

Comment:
I think this blog needs to be updated to reflect the changes in the implementation, i.e., it is dataSourceRef that is updated to include a namespace and there isn't a dataSourceRef2.

Reply:
Fixed.